css-two-column-layout/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column">
+        <h1 class="uppercase">Scale, Speed, Agility</h1>
+        <p>
+          We've dedicated thousands of man hours to making healthcare
+          institutions in the Helium network run 10x more efficiently and
+          deliver an unparalleled quality of care.
+        </p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-picture">
+        <img src="images/software.svg" alt="software">
+      </div>
+      <div class="column-half">
+        <h4 class="uppercase">Intuitive, Smart Software</h4>
+        <p>
+          We build products that transform how healthcare facilities run across Africa
+          -- making them radically more efficient optionally and finacially, a complete
+          turnaround.
+        </p>
+      </div>
```

Comment:
the one update you want here is to wrap both of these divs with one `column-half` div so that each row has two columns within it and all of the content is wrapped in a column box.